Generator.Infrastructure/IO/FileWriter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Configuration;
+using System.IO;
+using Generator.Application.Exceptions;
+using Generator.Application.Interfaces;
+
+namespace Generator.Infrastructure.IO
+{
+    public class FileWriter : IWriter
```

Comment:
zmienilbym nazwy, by bylo wyraznie widac, ze to writer i IWriter do zapisywania choice i picture